go.mod

```diff
@@ -43,7 +43,7 @@ require (
 	github.com/yandex-cloud/go-sdk v0.0.0-20241206142255-6c3760d17eea
 	github.com/youmark/pkcs8 v0.0.0-20240726163527-a2c0da244d78
 	go.uber.org/zap v1.27.0
-	golang.org/x/crypto v0.30.0
+	golang.org/x/crypto v0.31.0
```

Comment:
Huh, what prompted this change? 🤔

Reply:
The trivy scan: https://github.com/external-secrets/external-secrets/actions/runs/12329516819/job/34416450574#step:12:217

Reply:
Oh okay. Neat.